src/pages/index.js

```diff
@@ -13,72 +15,88 @@ import {
   profileTitleInput,
   profileDescriptionInput,
 } from "../utils/constants.js";
-
-// ---------------- Initial Cards Data ----------------
-// const initialCards = [
-//   {
-//     name: "Yosemite Valley",
-//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/yosemite.jpg",
-//   },
-//   {
-//     name: "Lake Louise",
-//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lake-louise.jpg",
-//   },
-//   {
-//     name: "Bald Mountains",
-//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
-//   },
-//   {
-//     name: "Latemar",
-//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
-//   },
-//   {
-//     name: "Vanoise National Park",
-//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
-//   },
-//   {
-//     name: "Lago di Braies",
-//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg",
-//   },
-// ];
-
-// ---------------- DOM Element References ----------------
-// const profileEditForm = document.forms["profile-form"];
-// const addCardForm = document.forms["card-form"];
-// const profileEditModal = document.querySelector("#profile-edit-modal");
-// const addCardModal = document.querySelector("#add-card-modal");
-// const previewImageModal = document.querySelector("#preview-image-modal");
-
-// const cardTitleInput = document.querySelector("#card-title");
-// const cardUrlInput = document.querySelector("#card-url");
-// const cardsListEl = document.querySelector(".cards__list");
-// const previewImage = previewImageModal.querySelector(".modal__image");
-// const previewCaption = previewImageModal.querySelector(".modal__caption");
-// const modalOpenedSelector = "modal_opened";
+import { data } from "jquery";
```

Comment:
`import { data } from "jquery";` should be deleted. 

* The project starts with errors, and nothing works https://snipboard.io/Bi8TwX.jpg